openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -104,3 +108,26 @@ def get_invalid(cls, instance):
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
+        output_node = instance.data.get("output_node")
+        rop_node = hou.node(instance.data["instance_node"])
+
+        frame = instance.data.get("frameStart", 0)
+        geo = output_node.geometryAtFrame(frame)
+
+        path_attr = rop_node.parm("path_attrib").eval() or "path"
+        paths = geo.findPrimAttrib(path_attr) and \
+                    geo.primStringAttribValues(path_attr)
+
+        if not paths:
+            output_node = instance.data.get("output_node")
+            path_node = output_node.parent().createNode("name", "AUTO_PATH")
+            path_node.parm("attribname").set(path_attr)
+            path_node.parm("name1").set('`opname("..")`/`opoutput(".", 0)`')
+
+            path_node.setFirstInput(output_node.input(0))
+            path_node.moveToGoodPosition()
+            output_node.setFirstInput(path_node)
+            output_node.moveToGoodPosition()
```

Comment:
Can we add a little logging here so it states what it created and where?

Reply:
Good point!, 
Thanks for reminding me